daemon/daemon.go

```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/docker/docker/daemon/exec"
 	"github.com/docker/docker/daemon/execdriver"
 	"github.com/docker/docker/daemon/execdriver/execdrivers"
-	"github.com/docker/docker/daemon/graphdriver"
 	_ "github.com/docker/docker/daemon/graphdriver/vfs" // register vfs
```

Comment:
On `make cross` daemon is not included for some platforms so these imports would not be active. But because of the `image.ID` type used by `reference` package the imports from `layer_store` would be active even for these platforms.